File: WS/Python/Q3.py
import datetime
import time

current_time = datetime.datetime.now()

print("Second :", current_time.second)
print("Minute : ", current_time.minute)
print("Hour : ", current_time.hour)
print("Day : ", current_time.day)
print("Month : ", current_time.month)
print("Year :", current_time.year)

print("weekday = ",current_time.weekday)
print("day of year = ",((current_time - datetime.datetime(current_time.year, 1, 1)).days + 1))
print("Current Date = ",current_time.day,"/",current_time.month,"/",current_time.year)
print("Current Time = ",(time.strftime("%H:%M:%S", time.localtime())))

File: WS/C/Q3.c
#include<stdio.h>
#include<time.h>

int main()
{
    time_t t = time(NULL);
    struct tm tm = *localtime(&t);
    printf("seconds = %d\n",tm.tm_sec);
    printf("Minutes = %d\n",tm.tm_min);
    printf("Hour = %d\n",tm.tm_hour);
    printf("day of month = %d\n",tm.tm_mday);
    printf("month of year = %d\n",tm.tm_mon+1);
    printf("year = %d\n",tm.tm_year+1900);
    printf("weekday = %d\n",tm.tm_wday);
    printf("day of year = %d\n",tm.tm_yday);
    printf("daylight savings = %d\n",tm.tm_isdst);
    printf("Current Date: %d-%02d-%02d\n",tm.tm_mday, tm.tm_mon + 1,tm.tm_year + 1900);
    printf("Current Time: %02d:%02d:%02d\n", tm.tm_hour, tm.tm_min, tm.tm_sec);
   
    
}